spec/factories/work_factory.rb

```diff
@@ -153,5 +156,37 @@
       end
     end
 
+    trait :oral_history_work do
+      title { "Oral history interview with William John Bailey" }
```

Comment:
FactoryGirl lets you have an additional factory (that inherits from the base work) in addition to traits. I'm never really sure when to use one with the other -- the main thing traits give you is the ability to combine them, but we'd never want to combine "oral history work" trait with, say, a "museum object" trait, so it _probably_ would work best as an additional factory instead of a trait. 

But not a big deal, and not a blocker to merge if you don't feel like doing it or it seems too confusing.